kobo/apps/trackers/tests/test_utils.py

```diff
@@ -39,6 +38,7 @@ def setUp(self):
         self.asset.deploy(backend='mock', active=True)
 
     def _create_product(self, product_metadata):
+        from djstripe.models import Price, Product
```

Comment:
Can't we use your decorator (`@requires_stripe`) here? 

If the answer is yes: 
I would like to see something like

```
Price = kwargs['product_model']
my_var = Price.objects.filter(...)
```
instead of calling `kwargs['product_model'].objects` directly 🙏